### PR TITLE
`Communication`: Increase the size of the question screen

### PR DESCRIPTION
**Pull request by canberkanar**
#### General

- [x] I tested **all** changes and their related features with **all** corresponding user types on a test server.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.cit.tum.de/dev/guidelines/language-guidelines/).
- [x] I chose a title conforming to the [naming conventions for pull requests](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/development-process.html#naming-conventions-for-github-pull-requests).
#### Client
- [x] I followed the [coding and design guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client/) and ensured that the layout is responsive.
- [x] I added multiple integration tests (Jest) related to the features (with a high test coverage), while following the [test guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/client-tests/).
- [x] I documented the TypeScript code using JSDoc style.
- [x] I added multiple screenshots/screencasts of my UI changes.

### Motivation and Context
The current container in which posts are listed violates our design principle (as implemented in the Exercises, Lectures and Statistics Tab) that the whole page should scroll. Solves #5906.

### Description
We remove the container for posts and instead allow the user to scroll the whole page to browse course posts.

### Steps for Testing

Prerequisites:
- 1 Student
- 1 Course with communication features enabled

1. Log in to Artemis
2. Navigate to Course Couumunication Overview
3. Make sure that posts are not displayed within a container and that the whole page can be scrolled
4. Make sure that the posts scroll without disruptions
5. Open your browser's developer tab and verify that items on the DOM tree are updated instead of appending new items as the user scrolls the page.

#### Code Review
- [x] Review 1
- [x] Review 2
#### Manual Tests
- [x] Test 1
- [x] Test 2

### Test Coverage

Not changed.

### Screenshots

https://user-images.githubusercontent.com/67344158/208880472-27035f20-667c-4e61-b90a-a15ae2448f96.mp4

### CODE REVIEW

**Review by pellumbbaboci — APPROVED** [review]
Tested on ts1, works as expected. Code changes look good to me.

**Comment by Vinicius-Agreste** [comment]
Tested on ts2, is working as expected. Expected behavior in the developer's tab as well.

**Review by Vinicius-Agreste — DISMISSED** [review]
Tested on ts2, is working as expected. Expected behavior in the developer's tab as well.

**Review by laadvo — APPROVED** [review]
Tested on ts1, virtual scrolling works well. Good improvement!

File: src/test/javascript/spec/component/virtual-scroll/virtual-scroll.component.spec.ts [hunk 1]
```diff
@@ -17,13 +17,17 @@ class MockRouter {
 describe('VirtualScrollComponent', () => {
```
Comment (ge65cer) [inline src/test/javascript/spec/component/virtual-scroll/virtual-scroll.component.spec.ts:17]: Unused import.

**Review by ge65cer — COMMENTED** [review]
Tested on TS2 on desktop and mobile, works. 👍

**Review by simon-hng — DISMISSED** [review]
Code

File: src/test/javascript/spec/component/virtual-scroll/virtual-scroll.component.spec.ts (hunk 1 above)
Comment (canberkanar) [reply src/test/javascript/spec/component/virtual-scroll/virtual-scroll.component.spec.ts:17]: Removed, thanks!

**Pull request merged**